### PR TITLE
MAINT: add Python 3.6 to appveyor, small edits

### PR DESCRIPTION
**Pull request by matthew-brett**
Add Python 3.6 and Windows 64 bit builds to appveyor.  A few small edits
for neatness.

As promised on issue gh-2839.

### CODE REVIEW

**Comment by cgohlke** [comment]
See https://github.com/scikit-image/scikit-image/pull/2846 and https://github.com/scikit-image/scikit-image/pull/2845 for Python 2.7 errors and crashes

**Comment by matthew-brett** [comment]
Good to go from my point of view.

File: .appveyor.yml [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
+    - PYTHON: C:\Python27
+    - PYTHON: C:\Python27-x64
+    - PYTHON: C:\Python35
+    - PYTHON: C:\Python35-x64
+    - PYTHON: C:\Python36
+    - PYTHON: C:\Python36-x64
 
-matrix:
-  fast_finish: true
+# matrix:
```
Comment (soupault) [inline .appveyor.yml]: Would you like to revert this?

**Comment by matthew-brett** [comment]
OK - reverted.

**Comment by soupault** [comment]
Errrmm... Some tests are failing, but AppVeyor keeps being green. We should investigate and fix this (in another PR, of course).

**Pull request merged**